earthpy/spatial.py

```diff
@@ -417,14 +428,16 @@ def hillshade(arr, azimuth=30, angle_altitude=30):
 
     Parameters
     ----------
-    arr: a numpy ndarray of shape (rows, columns) containing elevation values
-    azimuth:  default (30)
-    angle_altitude: default (30)
+    arr : numpy array
```

Comment:
Very nice. please add default values back and remove optional unless it is truly an optional parameter.